mir_eval/separation.py

```diff
@@ -422,6 +703,33 @@ def _bss_source_crit(s_true, e_spat, e_interf, e_artif):
     return (sdr, sir, sar)
 
 
+def _bss_image_crit(s_true, e_spat, e_interf, e_artif):
+    """Measurement of the separation quality for a given image in terms of
+    filtered true source, spatial error, interference and artifacts.
+
+    Parameters
```

Comment:
You don't need to include these in an underscore function if you aren't specifying them.  Where they appear elsewhere was due to some auto-docstring-formatting code that we run at some point.